038-MdwCovid19/Student/00-Background.md

```diff
@@ -0,0 +1,35 @@
+# Challenge 0: Background - CommonWealth of Caladan
```

Comment:
This challenge is not following the Challenge template from the author's guide.

Can the text use our template headers? 
https://github.com/microsoft/WhatTheHack/blob/master/000-HowToHack/WTH-Challenge-Template.md

Reply:
Gino & I chatted...  I'm okay to mark this one resolved if you don't wish to modify this one.